### DISCUSSION TITLE
Why lv 60 raids are important for wotlk?

### DISCUSSION THREAD

**Discussion by ypoptos** [chromiecraft/chromiecraft#3926] [General]
Why you waste months to make lv 60 raids in a lv 80 game? You plan to port it later to vanilla or tbc expansions? Ofc everyone wants 100% working game but be more realistic. This is a lv 80 expansion, why you need to fix the lv 60 and lv 70 raids. Ok, dungeons are useful for leveling, but raids are not.

**Comment by Stump876** [comment]
By this logic why not just start at level 80 and just drop everyone in Northrend? Most Pservers I've seen do something like this, or increase rates ignoring a lot of vanilla and BC content. But with AC, it's more about having a working 3.3.5 core with things operating as they would have in 3.3.5, starting at level 1- Including 60 and 70 raids. Different things matter to different people. I don't think it's unrealistic to have everything working as it should. It's a massive undertaking for sure, but really that's what brought all of us here. To help with it. Once it's finished you, me, or anyone, can take the core and open whatever server that you wish knowing that all aspects will be working as intended.

**Comment by 55Honey** [comment]
Hello Ypoptos,
thanks for reaching out.

To add to what Stump already said, you might want to look at our [Mission Statement](https://www.chromiecraft.com/en/about/):

>Most private servers will package up projects like TrinityCore or AzerothCore in wrapping paper and selfishly keep their development fixes to themselves despite using an open-source foundation to make their server. When the server inevitably dies, the fixes are lost with it (see [this article](https://github.com/FrancescoBorzi/why-I-hate-wow-private-servers/blob/master/ENGLISH.md)). ChromieCraft will be run by the AzerothCore team to benefit the open-source project. This means that all bug reports submitted and all fixes we make will go towards the development of open-source project AzerothCore. In addition to this, all donations will be put towards covering hosting costs and AzerothCore bounty programs.  This server will not be running for profit and there will be no donator pay-to-win mechanics. Not now, not ever. There may be donator rewards which do not affect the game such as titles, mounts, or pets. Thank you for joining us and make sure to have fun and enjoy your stay.

**Comment by Hacki95** [comment]
Another aspect I want to throw in is, think about it as "preservation". If we don't fix and document as much as we can NOW the information might be lost in a few years. I want to be able to play a FULL 3.3.5 experience even 20 years from now and the only way to archive this is by researching and fixing EVERYTHING we can.

**Comment by ypoptos** [comment]
You look the game at today's hybrid form. Now all the players are in hold at lv 60. If they want to play they are gearing their characters from dungeons, raids and some bgs. So you have geared lv 60 players to join raids. At this form the lv 60 raids have meaning. If you can port them to lower expansions maybe they have also meaning (because you fix the raids with wotlk talents and spells).
But look the game when you have available all the content. When you leveling and you reach 60 you don't have the gear to join raids. Not even the easier ones. And the truth is that most players at level 58 will start questing at Outland. Faster leveling and way better gear. Sadly, the same think will happen with lv 70 raids. I loved TBC (best expansion for me) and questing at Outland. And I hated that after tbc you couldn't enjoy tbc areas. 
With all the content available do you really think that anyone will join the lv 60 or lv 70 raids? Is it really worth the time and the effort to fix them?

**Reply by 55Honey** [reply]
> Is it really worth the time and the effort to fix them?

Summing up all the above replies, the short answer to this question is 100% yes.
